assets/src/edit-story/migration/migrations/v0018_defaultBackgroundElement.js

```diff
@@ -0,0 +1,63 @@
+/*
```

Comment:
@barklund A bit late but I just realized that Dashboard templates would likely need updating as well -- these are currently still using the previous non-migrated elements (if I understand correctly). At least I can still see `backgroundElementId` being used in the templates. cc @carlos-kelly

Reply:
Oh thanks, wasn't aware. @carlos-kelly, please chime in if we should keep templates updated as we migrate the data format?